src/Dhall/Core.hs

```diff
@@ -1207,6 +1222,8 @@ alphaNormalize (Note s e₀) =
     Note s e₁
   where
     e₁ = alphaNormalize e₀
+alphaNormalize (ImportAlt l₀ _r₀) =
```

Comment:
Shouldn't this be:

```haskell
alphaNormalize (ImportAlt l₀ r₀) =
    ImportAlt l₁ r₁
  where
    l₁ = alphaNormalize l₀
    r₁ = alphaNormalize r₀
```

Reply:
You commented [here](https://github.com/dhall-lang/dhall-haskell/pull/473#issuecomment-397667252):
> for normalization, normalize the first branch and return that

The point is that by the time we run `alphaNormalize` there should be no `ImportAlt` constructors anymore.
I can add a comment on that line so it looks less surprising.